lib/middleman-email/extension.rb

```diff
@@ -24,6 +24,7 @@ class Extension < Extension
       option :base_url, nil
       option :authentication, :plain
       option :build_before, true
+      option :local_only, true
```

Comment:
Why is this option true by default?

Reply:
I've already adjusted it in the next commit